src/main/java/org/opentripplanner/util/TravelOptionsMaker.java

```diff
@@ -26,11 +26,12 @@ public final class TravelOptionsMaker {
   }
 
   public static List<TravelOption> makeOptions(Graph graph, TransitService transitService) {
+    var service = graph.getVehicleParkingService();
     return makeOptions(
       transitService.getTransitModes(),
-      graph.hasBikeSharing,
-      graph.hasBikeRide,
-      graph.hasParkRide
+      false,
```

Comment:
Shouldn't this be read from the rental service?

Reply:
It probably should but beforehand it was always set to false as the code for setting it to true (extraction of rental stations from OSM) is long gone.

I'm happy to derive it from the service though.

Reply:
I think we can include that, even if it is a functional change